editor/code_editor.h

```diff
@@ -234,6 +227,8 @@ class CodeTextEditor : public VBoxContainer {
 	};
 	void convert_case(CaseStyle p_case);
 
+	void set_indent_using_spaces(const bool p_use_spaces);
```

Comment:
```suggestion
	void set_indent_using_spaces(bool p_use_spaces);
```
Primitive parameters don't need `const`.

Reply:
No idea how that got there. Will fix this plus the cpp.